### ISSUE TITLE
Basic build configuration support

### ISSUE THREAD

**Issue by JDECOMBE** [razzmatazz/csharp-language-server#54]
Hi !
I may have missed something but I feel like currently there is no way to handle Release/Debug configurations.
The main point of this request is for stuff like `#if DEBUG`.
I'm using this language server with NeoVim (lspconfig / csharp-ls) and it behaves only as a debug configuration. Looking into the code of this repository, I can't find an option to change the build configuration.
Is this something I've missed or is it not implemented ?

**Comment by razzmatazz** [comment]
Correct, this has not been implemented yet. I believe this is not very complicated to implement but I don't have much free time at my hands currently.

I can guide you if you want to make a PR
